BUILD

```diff
@@ -29,3 +29,35 @@ Maven repository for it. To build the binding:
         -Dpackaging=jar
 
 3. Uncomment `<module>nosqldb</module>` and run `mvn clean package`.
+
+# Mapkeeper 
+
+Mapkeeper binding doesn't get built by default because there is no
+Maven repository for it. To build the binding:
+
+1. follow "https://github.com/m1ch1/mapkeeper/wiki/Getting-Started" to Generate Thrift MapKeeper Binding
+   a.  install libevent
+   b.  install boost
+   c.  install thrift
+   d.  generate mapkeeper binding
+
+2. After generating mapkeeper binding, go the "mapkeeper/thrift/gen-java/target", and 
+   cp mapkeeper-1.1-SNAPSHOT.jar mapkeeper-1.0.jar 
+   mvn install:install-file -Dfile=mapkeeper-1.0.jar -DgroupId=com.yahoo.mapkeeper -DartifactId=mapkeeper -Dversion=1.0 -Dpackaging=jar
```

Comment:
I seems get the artifact issue in the latest version(0.12.0), after I run the "mvn clean package", I get the outputs:
[INFO] Reactor Summary:
[INFO] 
[INFO] YCSB Root .......................................... SUCCESS [  2.010 s]
[INFO] Core YCSB .......................................... SUCCESS [ 13.360 s]
[INFO] Per Datastore Binding descriptor ................... SUCCESS [  0.419 s]
[INFO] YCSB Datastore Binding Parent ...................... SUCCESS [  0.832 s]
[INFO] Accumulo DB Binding ................................ SUCCESS [06:10 min]
[INFO] Aerospike DB Binding ............................... SUCCESS [  7.636 s]
[INFO] ArangoDB Binding ................................... SUCCESS [  2.068 s]
[INFO] AsyncHBase Client Binding for Apache HBase ......... SUCCESS [01:01 min]
[INFO] Azure table storage Binding ........................ SUCCESS [  1.382 s]
[INFO] Cassandra 2.1+ DB Binding .......................... SUCCESS [ 52.470 s]
[INFO] Couchbase Binding .................................. SUCCESS [  2.999 s]
[INFO] Couchbase Java SDK 2.x Binding ..................... SUCCESS [  2.365 s]
[INFO] Azure DocumentDB Binding ........................... SUCCESS [  0.926 s]
[INFO] DynamoDB DB Binding ................................ SUCCESS [ 15.698 s]
[INFO] Elasticsearch Binding .............................. SUCCESS [ 19.951 s]
[INFO] Geode DB Binding ................................... SUCCESS [  8.180 s]
[INFO] Google Cloud Datastore Binding ..................... SUCCESS [  2.674 s]
[INFO] Google Cloud Bigtable Binding ...................... SUCCESS [  8.763 s]
[INFO] HBase 0.98.x DB Binding ............................ SUCCESS [  7.534 s]
[INFO] HBase 0.94.x DB Binding ............................ SUCCESS [  8.953 s]
[INFO] HBase 1.0 DB Binding ............................... SUCCESS [01:04 min]
[INFO] Hypertable DB Binding .............................. SUCCESS [  2.535 s]
[INFO] Infinispan DB Binding .............................. SUCCESS [  2.284 s]
[INFO] JDBC DB Binding .................................... SUCCESS [  4.903 s]
[INFO] Kudu DB Binding .................................... SUCCESS [  2.029 s]
[INFO] memcached binding .................................. SUCCESS [  1.775 s]
[INFO] MongoDB Binding .................................... SUCCESS [  4.154 s]
[INFO] Oracle NoSQL Database Binding ...................... SUCCESS [  1.138 s]
[INFO] OrientDB Binding ................................... SUCCESS [  8.265 s]
[INFO] rados of Ceph FS binding ........................... SUCCESS [  2.524 s]
[INFO] Redis DB Binding ................................... SUCCESS [  1.020 s]
[INFO] Rest Client Binding ................................ SUCCESS [ 27.616 s]
[INFO] Riak KV Binding .................................... SUCCESS [  6.584 s]
[INFO] S3 Storage Binding ................................. SUCCESS [  1.833 s]
[INFO] Solr Binding ....................................... SUCCESS [01:29 min]
[INFO] Solr 6 Binding ..................................... SUCCESS [01:45 min]
[INFO] Tarantool DB Binding ............................... SUCCESS [  1.562 s]
[INFO] YCSB Release Distribution Builder .................. FAILURE [  4.023 s]
[INFO] Mapkeeper DB Binding ............................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 15:23 min
[INFO] Finished at: 2016-12-21T16:11:39+08:00
[INFO] Final Memory: 124M/237M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-assembly-plugin:2.5.5:single (default) on project ycsb: Failed to create assembly: Artifact: com.yahoo.ycsb:mapkeeper-binding:jar:0.13.0-SNAPSHOT (included by module) does not have an artifact with a file. Please ensure the package phase is run before the assembly is generated. -> [Help 1]

How can I solve the problem?